cmd/psql/psql.go

```diff
@@ -74,7 +80,18 @@ You can pass additional arguments to psql after a double-dash (--):
 		databaseInstanceName := args[0]
 		extraArgs := args[1:]
 
-		return lakebase.Connect(cmd.Context(), databaseInstanceName, extraArgs...)
+		// Read retry configuration from flags
+		maxRetries, _ := cmd.Flags().GetInt("max-retries")
+
+		retryConfig := &lakebase.RetryConfig{
+			MaxRetries:    maxRetries,
+			InitialDelay:  time.Second,      // Fixed initial delay
+			MaxDelay:      10 * time.Second, // Fixed max delay
+			BackoffFactor: 2.0,              // Fixed backoff factor
+			Enabled:       maxRetries > 0,   // Disable retries when max-retries is 0
```

Comment:
Why have separate Enabled field if we already have MaxRetries=0 to indicate lack of retries?